android/src/test/java/io/keen/client/android/AndroidJsonHandlerTest.java

```diff
@@ -52,12 +68,17 @@ public void writeSimpleMap() throws Exception {
         map.put("boolean_property", true);
 
         // Serialize the map.
-        String result = serialize(map);
+        serialize(map);
```

Comment:
question: now that we're mocked, does this not return a real result? and that's why we're verifying the arguments instead?